website/docs/r/dashboard.html.markdown

```diff
@@ -683,6 +713,13 @@ Nested `widget` blocks have the following structure:
             - `log_query`: (Optional) The log query to use in the widget. The structure of this block is described [below](dashboard.html#nested-apm_query-and-log_query-blocks).
             - `process_query`: (Optional) The process query to use in the widget. The structure of this block is described [below](dashboard.html#nested-process_query-blocks).
             - `display_type` - (Optional) Type of display to use for the request. Available values are: `area`, `bars`, or `line`.
+            - `style` - (Optional) Style of the widget graph. One nested block is allowed with the following structure:
+              - `palette` - (Optional) Color palette to apply to the widget.
+              - `line_type` - (Optional) Type of lines displayed. Available values are: `dashed`, `dotted`, or `solid`.
+              - `line_width` - (Optional) Width of line displayed. Available values are: `normal`, `thick`, or `thin`.
+            - `metadata` - (Optional). Used to define expression aliases. Multiple nested blocks are allowed with the following structure:
+              - `expression` - (Required)
+              - `alias_name` - (Optional)
```

Comment:
Is there a reason one would add the expression but not an alias for it?

Reply:
probably bot a typical use case, but the UI allows it, and so does the the API and Go client.